.gitignore

```diff
@@ -85,4 +85,4 @@ compile_commands.json
 .envrc
 .DS_Store
 temp/
-/compile_commands.json
+/compile_commands.json
```

Comment:
Did you remove the tailing space for this line?

Reply:
There was no trailing space earlier. I dunno what it was, but its gone now :)